### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by elisha2kyakpo1**
This is the final project and I did the following;

1. Stying the whole project.
2. Made final changes.

### CODE REVIEW

File: src/components/Nav.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
 
 function Nav() {
   return (
```
Comment (ZawadiSumba66) [inline src/components/Nav.js:6]: - [ ] The `bookstore cms ` logo ought to be inline and not one below the other. Kindly fix this.
- [x] The book and categories links in the nav ought to use the `Montserrat` font family. Kindly also reduce the size of the fonts to match the given design.
- [x] The user icon positioned at the very end of the navbar is missing.

File: src/components/Book.js [hunk 2]
```diff
@@ -46,19 +44,21 @@ const Book = (props) => {
               </div>
             </div>
             <div className="percent">
-              <span className="circle" style={progress}> </span>
-              <div>
+              <span className="circle" style={progress}>
+                <div className="inner"> </div>
+              </span>
+              <div className="progress-contain">
```
Comment (ZawadiSumba66) [inline src/components/Book.js:48]: - [x] I believe the progress percentage is darker in color. Please refer to the design given.
- [x] I would also urge to you reduce its font-size as well

File: src/components/Book.js [hunk 3]
```diff
@@ -46,19 +44,21 @@ const Book = (props) => {
...
               </div>
             </div>
-
-            <div>
-              <h4>Current Chapter</h4>
-              <h2>{chapter}</h2>
+            <div className="border"> </div>
+            <div className="current-div">
+              <h4 className="current">Current Chapter</h4>
+              <h2 className="chapter">Chapter 17</h2>
```
Comment (ZawadiSumba66) [inline src/components/Book.js:59]: - [ ] Lets ensure there is some padding to the right of the book.Check the design below for comparison:
template's
image
==========================================================================
your's
image

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
### Hi @elisha2kyakpo1 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Nav.js (hunk 1 above)
Comment (elisha2kyakpo1) [reply src/components/Nav.js:6]: The icon is available, you may be missing something, with npm i, everything should be available

icon

Please let me know what you mean here "The bookstore cms logo ought to be inline and not one below the other. Kindly fix this.". These are for the icon from for font awesome. Check the screenshot about let me know what you mean

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi @elisha2kyakpo1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**